api4/plugin_test.go

```diff
@@ -305,11 +305,33 @@ func TestNotifyClusterPluginEvent(t *testing.T) {
 	require.Equal(t, "testplugin", manifest.Id)
 
 	// Successful remove
+	webSocketClient, err := th.CreateWebSocketSystemAdminClient()
+	require.Nil(t, err)
+	webSocketClient.Listen()
```

Comment:
Should we include a defer for `websocketClient.Close()` ?

Reply:
Great catch! Updated.